src/Microsoft.OpenApi/Properties/SRResource.resx

```diff
@@ -210,6 +210,12 @@
   <data name="Validation_RuleAddTwice" xml:space="preserve">
     <value>The same rule cannot be in the same rule set twice.</value>
   </data>
+  <data name="Validation_SchemaDoesntContainDiscriminatorProperty" xml:space="preserve">
+    <value>Schema {0} doesn't contain discriminator property {1}.</value>
+  </data>
+  <data name="Validation_SchemaDoesntContainDiscriminatorPropertyInRequiredFieldList" xml:space="preserve">
+    <value>Schema {0} doesn't contain discriminator property {1} in the required field list.</value>
```

Comment:
>Schema {0} doesn't contain discriminator property {1} in the required field list. [](start = 11, length = 81)

Composite schema {0} must contain property specified in the discriminator {1} in the required field list.